Notebooks/OOPS_TMT_Data_Processing_And_QC.Rmd

```diff
@@ -59,37 +77,45 @@ psm.f <- psm_data_flt[, setdiff(colnames(psm_data_flt), abundance_cols)]
 # update the column names to remove the 'Abundance.` prefix
 colnames(psm.e) <- gsub('Abundance.', '', colnames(psm.e))
 
-# we don't have 'phenotype' data to add so we just define the 
-# 'expression' data and 'feature' data
-
-psm.p <- data.frame(Treatment=rep(factor(c('Vehicle', 'Safe', 'Toxic','Vehicle', 'Safe', 'Toxic','Vehicle', 'Safe', 'Toxic','Vehicle', 'Safe', 'Toxic','Vehicle', 'Safe', 'Toxic')), c(1,1,1,1,1,1,1,1,1,1,1,1,1,1,1)), row.names=colnames(psm.e))
+# hardcoding experimental design as we know what it is
+psm.p <- data.frame(Treatment=factor(rep(c('Vehicle', 'Safe', 'Toxic'), times=5)), row.names=colnames(psm.e))
   
 psm <- MSnbase::MSnSet(exprs = psm.e, fData = psm.f, pData=psm.p)
+
+
+```
+
+We have some rows in our PSM quantification where all samples are NA. We want to remove these
+```{r}
+table(rowSums(is.na(exprs(psm))))
+psm <- psm[rowSums(is.na(exprs(psm))) < ncol(exprs(psm))]
+table(rowSums(is.na(exprs(psm))))
 ```
 
+
 Plot intensity distributions
 
 ```{r}
 plot_quant(log(psm, base=2), method='density')
 ```
 We want to remove low Signal:Noise (S:N) PSMs, since the quantification values will be less accurate and there will be more missing values. We can inspect the relationship between S:N and missing values using the plot_missing_SN function.
+
+
 ```{r}
-plot_missing_SN(psm, bins = 40)
+plot_missing_SN(psm[fData(psm)$Average.Reporter.SN<30,], bins = 20)
```

Comment:
Subsetting the PSMs to focus on those with low S:N. Makes it easier to assess suitable threshold.